types/types.go

```diff
@@ -116,6 +116,8 @@ const (
 	ControlPlaneName                 = "longhorn-manager"
 
 	DefaultRecurringJobConcurrency = 10
+
+	LonghornPVAnnotationVolumeStatus = "longhorn.io/volume-status"
```

Comment:
NIT: will this be more straightforward? `PVAnnotationLonghornVolumeSchedulingError = "longhorn.io/volume-scheduling-error"`